test/maps-search.test.ts

```diff
@@ -72,12 +72,26 @@ describe('Tomtom Places E2E Tests', () => {
             expect(res[0].country).toStrictEqual('Australia');
         });
 
-        it('handles error', async () => {
+        it('handles not a real address', async () => {
+            const fakeAddress = '!!%*^&%#$(&^';
+            expect(getPlaceAutocomplete(config.apiKey, fakeAddress)).rejects.toThrow('failed to retrieve data');
+        });
+
+        it('handles error when no address is passed', async () => {
```

Comment:
Additional test to handle when a fake or malformed address is used. Here I did try setting the fake address as "this is not a real address", however, it returned a result. To ensure my function worked as expected when nothing is returned from the tom-tom API, I used special characters for this test.